.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
+sudo: false
 node_js:
-  - 0.10
+  - 0.12
 
 branches:
   only:
     - master
 
 before_script:
-  - npm install -gq grunt-cli bower coveralls
-  - bower install
+  - npm install -gq coveralls
```

Comment:
Yup, exactly why I removed grunt-cli.  I removed coveralls as a dependency, so its installed globally here.  But to your point, since NPM scripts sources them, I should be able to get away without global here.